2.0.0 changes

cs-abhishek-shukla opened the pull request:
- release note conflicts
    - Configuration wizard in setup
    - SS change in Usage

cs-suyog-jinturkar commented on docs/setup.md:
```diff
@@ -22,11 +22,118 @@ Threat Intel Management (TIM) Solution Pack requires the following solution pack
+
+## Setup Threat Intel Management on FortiSOAR
+
+The option to launch the Threat Intel Management Configuration Wizard appears on the Threat Intel Management solution pack's content hub page.
```
Please include landing page with its SS and explain it here as well - since its another option to configure TIM (for the 1st time)

cs-abhishek-shukla replied:
Done

cs-suyog-jinturkar commented on docs/setup.md:
```diff
@@ -22,11 +22,118 @@ Threat Intel Management (TIM) Solution Pack requires the following solution pack
+
+## Configuring Feed Rules
+
+You can configure the following feed rules to manage threat intelligence feeds from various sources:
```
You might need to explain the why and how its beneficial for the user?  you can create separate .md file for this and refer it for feed  settings section as well

cs-abhishek-shukla replied:
Done

cs-suyog-jinturkar commented on docs/usage.md:
```diff
@@ -47,7 +47,7 @@ The *Feed Ingested* dashboard component reflects statistics for Threat Intel Fee
 Using a wide variety of feed integrations that are available on the **Content Hub**, you can seamlessly ingest feeds and get a normalized and aggregated view of the feeds on the **Threat Feeds** tab:  
 
-![Threat Feeds tab](res/threat-feeds.png)
+![Threat Feeds tab](res/threat-feeds.png) <!-- Change Screenshot -->
```
where are we explaining new landing page? navigation updates? first class dashboard in dashboard drop down?, new datasets with screenshots?, updated feedsettings page. etc.

cs-prerna-joshi commented on docs/setup.md:
```diff
@@ -22,11 +22,118 @@ Threat Intel Management (TIM) Solution Pack requires the following solution pack
+
+### Linking Threat Feeds to Indicators
+
+Enable this rule and specify a feed confidence threshold to automatically update the matching indicator record reputation. This rule updates the reputation of an indicator that matches a feed with the reputation of the feed. The indicator is updated only when the confidence of the feed is equal to or more than the value specified in the **Confidence Threshold** field (70% in our example):
```
If you set this value to 70, all feeds with a confidence level equal to or greater than 70 will link to the indicator.
We can write example in this format.

cs-abhishek-shukla replied:
Thanks. Done

cs-prerna-joshi commented on docs/setup.md:
```diff
@@ -22,11 +22,118 @@ Threat Intel Management (TIM) Solution Pack requires the following solution pack
+
+### Ingesting Unstructured Threat Feeds
+
+Enable this rule to ingest unstructured threat feeds from file sources and email communications. This rule extracts critical threat feeds from unstructured data sources and automatically ingests it into the Threat Feeds module.
```
```suggestion
Enable this rule to ingest unstructured threat feeds from file sources and email communications. This rule extracts critical threat feeds from unstructured data sources and automatically ingests it into the Threat Intel Feeds module.
```

cs-abhishek-shukla replied:
Done

cs-prerna-joshi commented on docs/setup.md:
```diff
@@ -22,11 +22,118 @@ Threat Intel Management (TIM) Solution Pack requires the following solution pack
+ 
+- **Ingest Threat Feeds from Email Attachments**: Select to ingest unstructured threat feeds from email attachments.
+	- **Email Client**: Select an email server from which to ingest emails. Currently, only *Exchange* is supported.
+	- **Email Folder** : Select the mailbox folder, containing threat feed attachments.
```
```suggestion
	- **Email Folder** : Specify the mailbox folder that contains all emails with feed attachments.
```

cs-abhishek-shukla replied:
Done

cs-prerna-joshi commented on docs/setup.md:
```diff
@@ -22,11 +22,118 @@ Threat Intel Management (TIM) Solution Pack requires the following solution pack
+	- **Email Folder** : Select the mailbox folder, containing threat feed attachments.
+
+> [!Note]
+> Attachments of only unread emails are ingested.
```
We can also add , 
1.You need to create separate folder for emails with feed attachments and need to redirect all such emails to this folder.
2. Please note only feeds from attachments will be extracted. Email body , subject line are not supported.

cs-abhishek-shukla replied:
Done

cs-sujeet-ahirrao commented on release_notes.md:
```diff
@@ -36,6 +40,9 @@ Managing threat data just got easier with these powerful updates:
+    - **Ingest Unstructured Threat Feeds**: Ingests unstructured feeds from files or email attachments
 
 These features enable seamless handling of diverse data formats and automated configurations, saving time and effort.
 
```
**Missing Points** 
- Added below connector settings at the time of installation 
   -  MITRE ATT&CK: Default configuration, Data Ingestion Playbook and Schedule
- Removed below connector settings at the time of installation 
   - Fortinet FortiGuard Threat Intelligence: Data Ingestion Playbook and Schedule (will create using wizard)
   - Malsilo: Removed from the SP installation 
 - Added description felid in Workspace details-layout

cs-abhishek-shukla replied:
Done

cs-sujeet-ahirrao commented on release_notes.md:
```diff
@@ -36,6 +40,9 @@ Managing threat data just got easier with these powerful updates:
+    - **Link Threat Feed to Indicators**: (Earlier *Feed to Indicator Linking*) Updates the indicator reputation as per the matching feed.
+    - **Ingest Unstructured Threat Feeds**: Ingests unstructured feeds from files or email attachments
 
 These features enable seamless handling of diverse data formats and automated configurations, saving time and effort.
```
After compilation of Wizard following changes happened
- Created `FortiGuard Outbreak Threat Feeds`, `FortiGuard Threat Intel Feeds` and `Phishing Threat Feeds` default datasets are created
- Change the `threat-intel-management-configuration`  key store record value from `Not Configured` to `Configured`
- MITRE data ingestion will trigger

cs-abhishek-shukla replied:
Not revealing information about the Key store record even if it is evident/obvious. Rest included

cs-dev-chaudhary commented on docs/setup.md:
```diff
@@ -22,11 +22,155 @@ Threat Intel Management (TIM) Solution Pack requires the following solution pack
+
+- The Exchange connector with email ingestion setup to ensure regular ingestion of threat feeds from email attachments.
+    - To configure and use the Exchange connector, refer to [Configuring Exchange](https://docs.fortinet.com/fortisoar/connectors/exchange).
+
```
We should mention Fortiguard connector as well if we are mentioning Fortirecon ACI and exchange.

cs-abhishek-shukla replied:
Exchange and FortiRecon ACI have to be manually configured. FortiGuard requires no input from user, wdyt?

cs-dev-chaudhary replied:
Yeah, that's correct—FortiGuard is configured only during the wizard launch. Later, when I was reading through some documentation, I saw that we're providing FortiGuard details as well. I thought we were listing all the connectors here, but forgot to mention FortiGuard so commented.

cs-dev-chaudhary commented on docs/setup.md:
```diff
@@ -22,11 +22,155 @@ Threat Intel Management (TIM) Solution Pack requires the following solution pack
+
+- **Ingestion Schedule**
+    - On the Scheduling screen, from the **Do you want to schedule the ingestion?** drop-down list, select **Yes**.
+    - Specify the `Cron` expression for the schedule. For example, if you want to pull data every 5 minutes, click **Hourly**. This means that the feeds are pulled and ingested every hour.
```
There is a discrepancy in the example, please check

cs-abhishek-shukla replied:
Fixed. Thanks

cs-dev-chaudhary commented on docs/setup.md:
```diff
@@ -22,11 +22,155 @@ Threat Intel Management (TIM) Solution Pack requires the following solution pack
+
+**Confidence Threshold**: Specify a feed confidence threshold to link feeds, with confidence threshold equal to or greater than the specified value, to the indicator.
+
+For example, if you set this value to 70, all feeds with a confidence level equal to or greater than 70 will link to the indicator.
```
And the reputation of the indicator( created after the existing feeds) will be updated.

cs-abhishek-shukla replied:
Note added

cs-dev-chaudhary commented on docs/setup.md:
```diff
@@ -22,11 +22,155 @@ Threat Intel Management (TIM) Solution Pack requires the following solution pack
+
+For example, if you set this value to 70, all feeds with a confidence level equal to or greater than 70 will link to the indicator.
+
+![Feed Configurations - General tab](./res/general.png)
```
The link is not working, please check

cs-abhishek-shukla replied:
Fixed. Thanks

cs-dev-chaudhary commented on docs/setup.md:
```diff
@@ -22,11 +22,155 @@ Threat Intel Management (TIM) Solution Pack requires the following solution pack
+    - **Email Folder** : Specify the mailbox folder that contains all emails with feed attachments.
+
+> [!NOTE]
+>- Attachments of only unread emails are ingested, hence we recommend that you create a separate folder for emails with feed attachments with mailbox rules in place to redirect all such emails to this folder.
```
We can provide an example for folder name like Unstructured Threat Feeds.

cs-abhishek-shukla replied:
I doubt people create folders by that name :-)

cs-dev-chaudhary commented on docs/setup.md:
```diff
@@ -22,11 +22,155 @@ Threat Intel Management (TIM) Solution Pack requires the following solution pack
+- **Email Ingestion Schedule**: Specify the frequency at which unstructured threat feeds are ingested from emails. This schedule will then automatically run at the specified frequency to ingest unstructured threat feeds. For example, if you want to ingest emails every 5 minutes, click **Every X Minute**, and in the **minute** box enter `*/5`. This means that emails are ingested every 5 minutes.
+    - **Timezone**: Select a timezone in which to export the report. Default is *`UTC`*.
+
+- **Block Threat Feeds Automatically**: Select this option to block threat feeds immediately on ingestion. Leave unchecked to manually block threat feeds.
```
Remove "Leave unchecked to manually block threat feeds."

cs-abhishek-shukla replied:
Removed. But why?

cs-dev-chaudhary replied:
Previously, we were providing a manual block PB, but we've now removed it from the collection. Donot want user have an impression we are providing manual PB as well.

cs-dev-chaudhary commented on docs/setup.md:
```diff
@@ -22,11 +22,155 @@ Threat Intel Management (TIM) Solution Pack requires the following solution pack
+
+### Content Hub Page
+
+After installation, the configuration wizard launches automatically.
```
Content.md needs to be updated , as malsilo is not default connector on TIM SP anymore.

cs-dev-chaudhary replied:
Threat Intel Management Configuration
TAXII Server Configuration
missing under widgets in content.md

cs-dev-chaudhary replied:
Playbook Collection
needs to be updated with new playbooks.

cs-dev-chaudhary replied:
Picklist in content.md
=================
AlertType
SIEMs

cs-abhishek-shukla replied:
This information needs to come from Dev/QA

cs-dev-chaudhary commented on docs/threat-feed-rules.md:
```diff
@@ -0,0 +1,74 @@
+    - Select the frequency at which unstructured threat feeds are ingested from emails. This schedule will then automatically run at the specified frequency to ingest unstructured threat feeds. For example, if you want to ingest emails every 5 minutes, click **Every X Minute**, and in the **minute** box enter `*/5`. This means that emails are ingested every 5 minutes.
+    - **Timezone**: Select a timezone in which to export the report. Default is *`UTC`*.
+
+5. **Block Threat Feeds Automatically**: Select this option to block threat feeds immediately on ingestion. Leave unchecked to manually block threat feeds.
```
Remove "Leave unchecked to manually block threat feeds."

cs-dev-chaudhary commented on release_notes.md:
```diff
@@ -25,7 +25,11 @@ This new navigation experience brings everything you need into a unified, intuit
 - **Mandatory Fields Compliance**: Feed modules now meet STIX/TAXII 2.0 requirements with key fields marked as mandatory.
 - **System View Template Update**: The System View Template (SVT) now includes new fields, ensuring comprehensive and up-to-date data representation.
-
+- **Feed Configuration Settings**: The sections of feed configuration settings have been rearranged under 3 tabs:
```
Two more type of authentication methods are introduced in feed configuration widget authentication
===========
Use API Key authentication (recommended). For more details refer to [Authentication API Guide](https://docs.fortinet.com/document/fortisoar/7.6.1/api-guide/797122/access-keys)
For tools that don't support passing custom headers in API calls, use X-API-KEY-<api_key_name> as the username and <api-key> as the password (recommended).

cs-abhishek-shukla replied:
The authentication methods were always present, just never mentioned. wdyt?

cs-dev-chaudhary commented on release_notes.md:
```diff
@@ -36,6 +40,23 @@ Managing threat data just got easier with these powerful updates:
 
 - **Unstructured Feeds Made Simple**: The new Import Unstructured Feeds button on the Threat Feed page lets you upload files and create feed records effortlessly.
 - **MITRE ATT&CK Connector with Default Configuration**: The MITRE ATT&CK connector now installs with a default configuration. You no longer need to manually configure it to ingest threat feeds from the MITRE organization.
```
Mitre point can be taken out from here, as these are not feeds.

cs-abhishek-shukla replied:
Mitre connector configuration is mandatory for running the scenario

cs-abhishek-shukla committed:
2.0.0 changes
    - release note conflicts
    - Configuration wizard in setup
    - SS change in Usage
    - Added a new file threat intel feeds
    - Updated contents.md
- Incorporated Sujit's feedback
